packages/playground/src/dashboard/components/user_nodes.vue

```diff
@@ -204,13 +204,16 @@ export default {
       try {
         loading.value = true;
         const twinId = profileManager.profile!.twinId;
+        console.log("Twin ID", twinId);
 
         const { data, count } = await gridProxyClient.nodes.list({
           retCount: true,
           page: page.value,
           size: pageSize.value,
-          rentedBy: twinId as number,
+          ownedBy: twinId as number,
```

Comment:
I think you might need to merge development since this change is already present on development branch.

https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/playground/src/dashboard/components/user_nodes.vue#L210

Reply:
merged